README.md

```diff
@@ -53,6 +54,16 @@
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo" href="https://pmaxy.github.io/setup-and-mobile-first/" ></a>
+
+> Add a link to your deployed project.
+
+- [https://pmaxy.github.io/setup-and-mobile-first/]
```

Comment:
[Optional] In order to properly markup the link and not necessary show the full URL in text, you may use the following text for the line above:

Markup:
![image](https://user-images.githubusercontent.com/20554583/214494840-60f45def-fda5-406a-99fb-de69a0d17f27.png)

Display:
![image](https://user-images.githubusercontent.com/20554583/214494950-ced6edd7-303b-4b5e-8e88-5a804a87f1f2.png)

Reply:
thank you alot i will make the changes